shae-riches/week5/wed/the-company-v4/config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  
+  root to: 'pages#home'
```

Comment:
this has given you an error `uninitialized constant PagesController`
What is this trying to tell you?
Do you have a PagesController?
You have set up a route, now...?
https://kristabel-wong.gitbook.io/sei55-git-book/week-5/day-22-rails-with-db#2.-set-up-routes-in-the-routes.rb-file